xenia-api/src/main/groovy/pl/jug/torun/xenia/oauth/OAuthData.groovy

```diff
@@ -0,0 +1,41 @@
+package pl.jug.torun.xenia.oauth
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+import org.springframework.stereotype.Component
+
+import javax.annotation.PostConstruct
+import java.awt.Desktop
+
+@Component
+@Configuration
+@ConfigurationProperties(prefix = "oauth")
+class OAuthData {
+    String requestToken
+    String refreshToken
+    int expiryPeriod
+
+    String clientId
+    String clientSecret
+
+    @PostConstruct
+    void init() {
+        String address = "https://secure.meetup.com/oauth2/authorize" +
+                "?client_id="+ clientId +
+                "&response_type=code" +
+                "&redirect_uri=http://localhost:8080/oauth2/redirect";
+
+        if (Desktop.isDesktopSupported() && Desktop.getDesktop().isSupported(Desktop.Action.BROWSE)) {
```

Comment:
I understand the shortcut, but this is rather bad UX, I think exposing this on a separate endpoint would be beneficial. Then UI can slurp it, redirect and after successful access can redirect back to app seamlessly.
This scenario is on my TODO list.